commcare_connect/connect_id_client/main.py

```diff
@@ -81,6 +81,12 @@ def fetch_user_counts() -> dict[str, int]:
     return data
 
 
+def fetch_non_invited_user_signup_dates() -> dict[str, int]:
+    response = _make_request(GET, "/users/fetch_non_invited_user_signups")
```

Comment:
Is it possible to combine this with the other call being made to connectid when generating the KPI table, so that we only have to make a single external network call.

Reply:
https://github.com/dimagi/commcare-connect/pull/815/commits/e5cc348e5ff03545e7afaf7b9b9710c605afebeb